apps/qubit/modules/actor/actions/browseAction.class.php

```diff
@@ -578,6 +578,13 @@ public function execute($request)
 
     $this->populateAggs($resultSet);
 
+    // Default to hide the advanced search panel
+    $this->showAdvanced = false;
+    if (filter_var($request->showAdvanced, FILTER_VALIDATE_BOOLEAN))
```

Comment:
Could you just do `$this->showAdvanced = filter_var($request->showAdvanced, FILTER_VALIDATE_BOOLEAN)`?

Reply:
Why yes, you can! Very elegant @mcantelon. 👍